bedrock/mozorg/tests/test_helper_misc.py

```diff
@@ -593,3 +598,24 @@ def test_firefox_ios_url_param(self):
             '/app/apple-store/id989804926?pt=373246&amp;mt=8&amp;ct=mozorg')
         eq_(self._render('es-ES', 'mozorg'), 'https://itunes.apple.com/es'
             '/app/apple-store/id989804926?pt=373246&amp;mt=8&amp;ct=mozorg')
+
+
+@override_settings(FXA_IFRAME_SRC_DIST_MAP=TEST_FXA_IFRAME_SRC_DIST_MAP)
+class TestFxAIFrameSrc(TestCase):
+    rf = RequestFactory()
+
+    def _render(self, path='/'):
+        req = self.rf.get(path)
+
+        return render("{{ fxa_iframe_src() }}", {'request': req})
+
+    def test_fxa_iframe_src_no_dist(self):
+        eq_(self._render('/'), 'https://accounts.firefox.com/')
```

Comment:
Since different environments need different values for the FxA iframe src, could we use `settings.FXA_IFRAME_SRC` in place of the hard-coded FxA production URL?